jujugui/static/gui/src/app/init.js

```diff
@@ -719,6 +719,12 @@ class GUIApp {
       */
       this.stripe = new window.jujulib.stripe(
         'https://js.stripe.com/', config.stripeKey);
+      /**
+        Application instance of the rates api.
+        @type {Object}
+      */
+      this.rates = new window.jujulib.rates(
+        config.ratesURL, new WebHandler());
```

Comment:
So the rates service is not the same as plans, correct?

Reply:
Anyway, if we need this new config then we also need to update the Python files for the WSGI app.

Reply:
Yes, good catch, I somehow missed to commit those changes.